thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detection/algorithm/MovingWindowAlgorithmTest.java

```diff
@@ -21,9 +21,6 @@
 import java.util.List;
```

Comment:
Looks like a unrelated file change.

Reply:
Good catch. I wonder if it's worth making a separate PR though

Reply:
Let's just put it here. Thank you for cleaning up the tests!